src/main/java/edu/tulane/cs/hetml/vision/CLEFImageReader.java

```diff
@@ -68,13 +70,16 @@ public CLEFImageReader(String directory, String trainFilePath, String testFilePa
         testSegments = new ArrayList<>();
         testRelations = new ArrayList<>();
 
+        // all Segment
+        allSegments = new ArrayList<>();
+
         path = directory;
         // Load redefined segment relations
-        getRedefinedRelations(directory);
+//        getRedefinedRelations(directory);
```

Comment:
We need to load relations, no?

Reply:
No, these are redefined relations, we need the original ones

Reply:
And where we do load them?

Reply:
getallImages(directory);